packages/automerge-repo/test/DocHandle.test.ts

```diff
@@ -72,6 +72,21 @@ describe("DocHandle", () => {
     assert.equal(doc?.foo, "bar")
   })
 
+  it("no pending timers after a document is loaded", async () => {
```

Comment:
I feel like this test is a little brittle because it's entirely possible we would add other timers - e.g. something which announces presence every so often on a DocHandle. I understand why this test is here though, we don't currently have an API for saying "everything stop now". I think we should just add a comment that once we have a `Repo.stop` API and accompanying tests then this test will no longer be necessary.